# Cell type-specific Gene set enrichment analysis -mremaR

![image](https://github.com/user-attachments/assets/db0b0479-0a3e-466d-9c47-a29cfddd22cf)

- In the above figure, we have considered an example of blood RNAseq; therefore, the cellular proportion of blood will be considered. 
- Deconvolution algorithms providing cellular proportions per sample - 
CIBERSORTx, Iced-T, Immunedeconv, MuSiC, xCell, Blender, dTangle, DeconRNAseq etc. 
https://liulab-dfci.github.io/RIMA/Infiltration.html#cibersort

- Cell type-specific differential expression calling algorithms - 
CARseq, TOAST, CeDAR, CellDMC, LRCDE, TCA, csSAM, HIRE, and DESeq2, all of which are available as R packages.

- Cell type-specific Gene set analysis - 
mremaR - https://github.com/osedo/mremaR


If you have any questions, you can email us:
m.chopra1@universityofgalway.ie / d.oshea20@universityofgalway.ie (DÃ³nal O'Shea wrote mremaR)


